vendors/espressif/boards/esp32/aws_demos/application_code/espressif_code/mbedtls/CMakeLists.txt

```diff
@@ -1,21 +1,43 @@
-set(AMAZON_FREERTOS_ABSTRACTIONS_DIR "${AFR_MODULES_ABSTRACTIONS_DIR}")
-set(AMAZON_FREERTOS_3RDPARTY_DIR "${AFR_3RDPARTY_DIR}")
+register_component()
```

Comment:
Looks good to me, you also need this in aws_tests project right?

Reply:
Yes. I've updated the corresponding file under aws_tests as well.